google/cloud/pubsub/samples/client_samples.cc

```diff
@@ -161,12 +161,13 @@ void TopicAdminClientSetEndpoint(std::vector<std::string> const& argv) {
     throw examples::Usage{"topic-admin-client-set-endpoint"};
   }
   //! [topic-admin-client-set-endpoint]
```

Comment:
We should delete these handwritten samples and point https://github.com/googleapis/google-cloud-cpp/blob/115a348560952928e13ed3ad2f28488a3d827ba4/google/cloud/pubsub/doc/pubsub-main.dox#L123 to https://github.com/googleapis/google-cloud-cpp/blob/115a348560952928e13ed3ad2f28488a3d827ba4/google/cloud/pubsub/admin/samples/topic_admin_client_samples.cc#L40

for all of the generated admin snippets.

Reply:
Does deleting the handwritten ones need to be done separately?

Reply:
No, I would delete the Topic/Subscription admin client samples in this PR.

Reply:
Done